src/main/java/teammates/storage/api/StudentsDb.java

```diff
@@ -90,11 +90,15 @@ public void deleteDocument(StudentAttributes studentToDelete){
     }
     
     public void createStudents(Collection<StudentAttributes> studentsToAdd) throws InvalidParametersException{
+        
+        // Currently this createStudents function is only used in testing process
         List<EntityAttributes> studentsToUpdate = createEntities(studentsToAdd);
         for(EntityAttributes entity : studentsToUpdate){
             StudentAttributes student = (StudentAttributes) entity;
             try {
-                updateStudentWithoutDocument(student.course, student.email, student.name, student.team, student.section, student.section, student.googleId, student.comments);
+                
+                //This function is only used for testing as its purpose is to not create document if not necessary.                 
+                updateStudentWithoutDocument(student.course, student.email, student.name, student.team, student.section, student.email, student.googleId, student.comments);
```

Comment:
That means the student record is not added to the search index. The
"document" is the data structure used to store data in the search index.
'No document' means 'not searchable'. It's a shortcut we take to make tests
run faster.